CommunityToolShedMvc/Views/Community/Index.cshtml

```diff
@@ -28,14 +29,19 @@
                 <td scope="col">@item.Usage</td>
                 <td scope="col">@item.Warning</td>
                 <td scope="col">@item.Age</td>
-                @if (((CommunityToolShedMvc.Security.CustomPrincipal)User).Person.Id == item.OwnerId)
+                @if (((CommunityToolShedMvc.Security.CustomPrincipal)User).Person.Id == item.OwnerId ||
+                    ((CommunityToolShedMvc.Security.CustomPrincipal)User).IsInRole(Model.Id, "Reviewer"))
```

Comment:
I like how you're only displaying the "Remove" when the user can actually remove an item, but you should be checking these permissions in the `Tool.Remove` POST action method to ensure that the current has the appropriate permissions.